### PR TITLE
Added "Media player volume slider" card feature.

### PR DESCRIPTION
**Pull request by simon-zumbrunnen**
## Proposed change

Adds the possibility to add a "Media player volume" feature (slider) to the "Tile card".

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
- type: tile
  entity: media_player.living_room
  features:
    - type: "media-player-volume"
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/card-features/hui-media-player-volume-card-feature.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import { isUnavailableState } from "../../../data/entity";
+import type { HomeAssistant } from "../../../types";
+import type { LovelaceCardFeature } from "../types";
+import { cardFeatureStyles } from "./common/card-feature-styles";
+import type { MediaPlayerCardFeatureConfig } from "./types";
+
+export const supportsMediaPlayerVolumeCardFeature = (stateObj: HassEntity) => {
+  const domain = computeDomain(stateObj.entity_id);
+  return domain === "media_player";
+};
```
Comment (MindFreeze) [inline src/panels/lovelace/card-features/hui-media-player-volume-card-feature.ts:16]:
```suggestion
import { MediaPlayerEntityFeature } from "../../../data/media-player";
import { supportsFeature } from "../../../common/entity/supports-feature";

export const supportsMediaPlayerVolumeCardFeature = (stateObj: HassEntity) => {
  const domain = computeDomain(stateObj.entity_id);
  return (
    domain === "media_player" &&
    supportsFeature(stateObj, MediaPlayerEntityFeature.VOLUME_SET)
  );
};
```
Some players (like Chromecast) don't support volume

**Review by MindFreeze — CHANGES_REQUESTED** [review]

**Comment by MindFreeze** [comment]
TBH, I prefer buttons for volume but this is still an awesome feature

**Comment by simon-zumbrunnen** [comment]
> TBH, I prefer buttons for volume but this is still an awesome feature

You might not be the only one. Would it make sense to call it "Media player volume *slider*" so a second feature called "Media player volume *buttons*" could be created in the future?

**Comment by MindFreeze** [comment]
Yeah, good idea

**Review by MindFreeze — DISMISSED** [review]

**Review by MindFreeze — APPROVED** [review]

**Pull request merged**